ISSUES.md

```diff
@@ -0,0 +1,57 @@
+# Issues
+
+## How to create new issues properly
+
+GitHub issues are used for everything from bug reporting to long-term ideas. As such you can make everything much smoother by following some simple rules.
+
+### Rule zero
+
+Always try to give your issue a meaningful title as this is the first thing anyone will see.
+
+Note: `[CR]` and `[WIP]` "tags" are meaningful only for PRs. All open issues by definition are request for comments and work in progress.
+
+### Bug reports
+
+Before you submit a bug always search the issues to see if it hasn't been reported already.
+
+Your bug report has to include:
+
+- On what OS did you experience the problem (Windows, Linux, OS X etc.)
+- What version were you playing:
+ - Tiles or Curses (text-based)
+ - Version string (preferably full version e.g. "0.C-4547-g3f1c109", or Jenkins build number e.g. 3245)
```

Comment:
Yeah I don't like build numbers too so I did:
```
09:07:19 <       dRbiG> jenkins about 3245
09:07:19 <  CopperTube> #3245 SUCCESS "Started by GitHub push by Coolthulhu"
09:07:19 <  CopperTube> culprits: trinsic; at g973e82e on 2015-06-03 02:13:59 UTC
```
I could remove or even specify "jenkins build number is *not a version*", but would that actually work? I can make a webhook to translate the build number to git commit at which the build has been made easily. Or both.